frontend/src/__mocks__/mockServingRuntimesConfig.ts

```diff
@@ -0,0 +1,78 @@
+import { ConfigMapKind } from '~/k8sTypes';
+
+type MockResourceConfigType = {
+  name?: string;
+  namespace?: string;
+};
+
+export const mockServingRuntimesConfig = ({
+  name = 'servingruntimes-config',
+  namespace = 'test-project',
+}: MockResourceConfigType): ConfigMapKind => ({
+  kind: 'ConfigMap',
+  apiVersion: 'v1',
+  metadata: {
+    name,
+    namespace,
+    uid: 'e581ed91-f36d-4826-85cf-2526c61028b3',
+    resourceVersion: '801334675',
+    creationTimestamp: '2022-09-16T17:02:55Z',
+    labels: {
+      app: 'odh-dashboard',
+      'app.kubernetes.io/part-of': 'odh-dashboard',
+    },
+    annotations: {
+      'kfctl.kubeflow.io/kfdef-instance': 'odh-core.odh',
+      'kubectl.kubernetes.io/last-applied-configuration':
```

Comment:
Same with last-applied-configuration, seems a little weird for me, but it's not bad.